ccf_specs.md

```diff
@@ -576,72 +609,104 @@ composite-type = [
     ]
 ]
 
+composite-interface-type = [
```

Comment:
Maybe rename this to just `interface-type`, like it is used in Cadence

Reply:
Good idea!  Done in commit 0e2b78486e46507f3a1bf59551d5ba003eb75a42.